kcbq-connector/src/test/java/com/wepay/kafka/connect/bigquery/config/BigQuerySinkConfigTest.java

```diff
@@ -102,6 +104,54 @@ public void testTablesToTopics() {
   }
 
   @Test
+  public void testTopicsToTables() {
```

Comment:
A test that validates that multiple groups in a single regex work would be good.